apps/meteor/client/hooks/useUserCustomFields.ts

```diff
@@ -27,6 +27,11 @@ export const useUserCustomFields = (customFields: CustomField): CustomFieldDispl
 		return undefined;
 	}
 
+	if (!Array.isArray(customFieldsToShowObj)) {
```

Comment:
Is there any possibility of this value being different than an array? Like booleans or numbers? Should we validate it too?

Reply:
The only type of data that should be accepted is an array, so I don't think we should validate anything other that if the data is an array here, since in the following map we already have some checks for valid values